tests/test_wave_03.py

```diff
@@ -54,13 +54,14 @@ def test_friends_unique_movies_not_duplicated():
 
     # Assert
     assert len(friends_unique_movies) == 3
-
-    raise Exception("Test needs to be completed.")
+    assert HORROR_1 in friends_unique_movies
+    assert FANTASY_4 in friends_unique_movies
```

Comment:
Good job here! You could also add the assertion to check for the 3rd movie as well.